scenes/MosaikHeer.scn

```diff
@@ -35,189 +35,189 @@
 /ch/01/preamp -18.0 OFF ON 24  42
 /ch/01/gate OFF GATE -56.0 42.0 1 5.02  25 0
 /ch/01/gate/filter ON 3.0 85.3
-/ch/01/dyn ON COMP PEAK LOG -22.0 5.0 2 4.00 63 0.02  55 POST 0 100 OFF
+/ch/01/dyn ON COMP PEAK LOG -13.0 10 0 4.00 66 0.02  55 POST 0 75 OFF
```

Comment:
Do you think, we should keep the values for dynamics and EQs as in the previous version, or should we take the current one? Why did those changes happen? What do you think was the reason for the change? Did we change the acoustic setup of the drums?

Reply:
We can keep the previous values. Since there are a lot of variables in play when it comes to drums (heavy vs. light hitter, choice of drum sticks, dampening etc., for compressor shown here also gain setting of channel) those would have to be adjusted every Sunday anyway.

Regarding the setup of the drums: I find myself using the drum tortillas on snare only, even with heavy hitters. But previous settings should work for this, too.

Reply:
Okay, I will roll back to the settings of version 4.4.0 after we had the drum calibration session.